Userland/Libraries/LibCore/System.cpp

```diff
@@ -1172,9 +1172,19 @@ ErrorOr<void> join_jail(u64 jail_index)
     HANDLE_SYSCALL_RETURN_VALUE("jail_attach", rc, {});
 }
 
-ErrorOr<u64> create_jail(StringView jail_name)
+ErrorOr<u64> create_jail_with_default_isolation_options(StringView jail_name)
 {
-    Syscall::SC_jail_create_params params { 0, { jail_name.characters_without_null_termination(), jail_name.length() } };
+    Syscall::SC_jail_create_params params { 0, { jail_name.characters_without_null_termination(), jail_name.length() }, 1 << 0 };
```

Comment:
In terms of using the jails API from userspace, it would be nice to have the flags available here as well instead of constants like `1 << 0`.